x-pack/legacy/plugins/siem/public/containers/case/constants.ts

```diff
@@ -7,11 +7,3 @@
 export const CASES_URL = `/api/cases`;
 export const DEFAULT_TABLE_ACTIVE_PAGE = 1;
 export const DEFAULT_TABLE_LIMIT = 5;
-export const FETCH_FAILURE = 'FETCH_FAILURE';
```

Comment:
why do we switch from variables to inline strings?

Reply:
to make the `Action` type in the hooks better

Reply:
cannot we use `ActionCreator` from `typescript-fsa` as in the rest of the app?

Reply:
<img width="785" alt="Screen Shot 2020-03-10 at 11 48 05 AM" src="https://user-images.githubusercontent.com/6935300/76342984-121a1500-62c5-11ea-8955-e83e47de9967.png">
It doesn't resolve the type/value issue. take a look at the types in `use_get_cases.tsx`. do you think i should refactor to `ActionCreator`?